modules/nw-egress-router-about.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * networking/egress_router/using-an-egress-router.adoc
+
+[id="nw-egress-router-about_{context}"]
+= About an egress router
+
+The {product-title} egress router runs a service that redirects traffic to a
```

Comment:
Nitpick: Techincally it's  a pod, a more precis wording would be:

The {product-title} egress router runs a pod usually accessed from a service

Reply:
I've updated all instances of egress router to _egress router pod_.